fdb-record-layer-core/src/test/java/com/apple/foundationdb/record/provider/foundationdb/query/FDBRecordStoreQueryTestBase.java

```diff
@@ -417,7 +417,7 @@ protected <T> List<T> fetchResultValues(FDBRecordContext context, RecordQueryPla
     }
 
     protected <T> List<T> fetchResultValues(FDBRecordContext context, RecordQueryPlan plan, Function<Message, T> rowHandler,
-                                        TestHelpers.DangerousConsumer<FDBRecordContext> checkDiscarded, ExecuteProperties executeProperties) throws Exception {
+                                            TestHelpers.DangerousConsumer<FDBRecordContext> checkDiscarded, ExecuteProperties executeProperties) throws Exception {
```

Comment:
revert.

Reply:
So I think we actually want the white space here. The original code looks like this: https://github.com/FoundationDB/fdb-record-layer/blob/main/fdb-record-layer-core/src/test/java/com/apple/foundationdb/record/provider/foundationdb/query/FDBRecordStoreQueryTestBase.java#L419 the indentation doesn't look correct.

Reply:
Got it, thank you.